### ISSUE TITLE
jQuery broken in all pages due to missing version number in URL

### ISSUE THREAD

**Issue by EricDunsworth** [wet-boew/GCWeb#986]
@LaurentGoderre Could this be related to some of PR #983's changes?

In all of GCWeb's demo pages, the script references to jQuery are currently showing up like this:
```
<script src="http://ajax.googleapis.com/ajax/libs/jquery//jquery.min.js"></script>
```

They should be this:
```
<script src="http://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
```

**PS:** I'm not currently having any luck setting up GCWeb's build system locally, so I haven't been able to try fixing it.

**Comment by LaurentGoderre** [comment]
The next fixed seems like it fixes this...

**Comment by LaurentGoderre** [comment]
Fixed by #985
